web-jehlomat/src/screens/SeznamUzivatelu/SeznamUzivatelu.tsx

```diff
@@ -49,23 +51,37 @@ const ListInfo = styled.div`
     color: ${grey};
 `;
 
-const SeznamUzivatelu: FC<Props> = () => {
+const buttonStyles = {
+    marginLeft: '10px',
+    marginBottom: '0px',
+    color: `${primaryDark}`,
+    textDecoration: 'underline'
+}
+
+const SeznamUzivatelu: FC<Props> = (props: any) => {
     const [users, setUsers] = useState(usersMock);
     const listTitle = `${users.length} uživatel${users.length > 0 ? (users.length === 1 ? '' : users.length < 5 ? 'é' : 'ů') : 'ů'}`;
+    const isMobile = useMediaQuery(media.lte('mobile'));
+    let history = useHistory();
```

Comment:
```suggestion
    const history = useHistory();
```
Just to be consistent with what's already in place.